File: helloworld.py
#!/usr/bin/python
import sys

class Hello:
	name="fajoy"
	def __init__(self,name):
		self.name=name

	def show(self,text):
		if text:
			print self.name +":" +text
		else:
			print self.name + ":Hello World!!"

if __name__ == "__main__":
	s=Hello("fajoy")
        if len(sys.argv)>1:
		s.show(sys.argv[1])
	else:
		s.show("")

File: helloworld.c
#include<stdio.h>
 
int main(int argc,char *argv[],char *envp[])
{
   printf("Hello World\n");
   
   int i=0;
   for(i=0;i<argc;i++){
	printf("arg[%d]=%s\n",i,argv[i]);
   }
   i=0;
   while(envp[i]){
	printf("envp[%d]=%s\n",i,envp[i]);
	i++;
   }

   return 0;
}


